src/coreclr/src/vm/interoplibinterface.cpp

```diff
@@ -18,10 +18,13 @@ using CreateComInterfaceFlags = InteropLib::Com::CreateComInterfaceFlags;
 namespace
 {
     // This class is used to track the external object within the runtime.
+    // Keep in sync with ExternalObjectContextInterface in request.cpp
     struct ExternalObjectContext
     {
+        static constexpr intptr_t ExternalObjectContextSentinelValue = (intptr_t)0xE0E0E0E0E0E0E0E0;
```

Comment:
Perhaps move the sentinel value into the header file - `src/coreclr/src/vm/interoplibinterface.h`?

Reply:
Sure, then I can use the same definition in the dac too